### PR TITLE
Regenerate SDK

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary

This PR adds the newest generated code using API definition commit `433f7fd45da5addb51abddb2616ec805aeb24da6` and generator commit `a5d0b22673bd91f2802b06bf39e5b049501dcb18`.

Additionally generation templates were revised to fix some post-generation hand written changes.

### CODE REVIEW

File: src/IBM.WatsonDeveloperCloud.SpeechToText.v1/SpeechToTextService.cs [hunk 1]
```diff
@@ -2740,30 +2778,40 @@ public BaseModel TrainAcousticModel(string customizationId, string customLanguag
...
+        /// <param name="force">If `true`, forces the upgrade of a custom acoustic model for which no input data has
+        /// been modified since it was last trained. Use this parameter only to force the upgrade of a custom acoustic
+        /// model that is trained with a custom language model, and only if you receive a 400 response code and the
+        /// message `No input data modified since last training`. See [Upgrading a custom acoustic
+        /// model](https://cloud.ibm.com/docs/services/speech-to-text/custom-upgrade.html#upgradeAcoustic). (optional,
+        /// default to false)</param>
         /// <param name="customData">Custom data object to pass data including custom request headers.</param>
         /// <returns><see cref="BaseModel" />BaseModel</returns>
-        public BaseModel UpgradeAcousticModel(string customizationId, string customLanguageModelId = null, Dictionary<string, object> customData = null)
+        public BaseModel UpgradeAcousticModel(string customizationId, string customLanguageModelId = null, bool? force = null, Dictionary<string, object> customData = null)
```
Comment (lpatino10) [inline src/IBM.WatsonDeveloperCloud.SpeechToText.v1/SpeechToTextService.cs]: Probably a silly question considering how many times I've reviewed this SDK, but adding the parameter here doesn't break anything, right? I know you can use named parameters in C# but wasn't sure if someone could pass in `customData` as the last param without specifying the name.

Comment (mediumTaj) [reply src/IBM.WatsonDeveloperCloud.SpeechToText.v1/SpeechToTextService.cs]: Yeah this is technically a breaking change if they don't use named params. Let me move this after `customData`

**Comment by mediumTaj** [comment]
@lpatino10 Finally passed CI. Can you take a look again?

**Comment by mediumTaj** [comment]
no all good! thanks for keeping me honest!

**Pull request merged**